modules/migration-mtc-release-notes-1-8-5.adoc

```diff
@@ -0,0 +1,65 @@
+// Module included in the following assemblies:
+//
+// * migration_toolkit_for_containers/mtc-release-notes.adoc
+:_mod-docs-content-type: REFERENCE
+[id="migration-mtc-release-notes-1-8-5_{context}"]
+= {mtc-full} 1.8.5 release notes
+
+[id="technical-changes-1-8-5_{context}"]
+== Technical changes
+
+{mtc-first} 1.8.5 has the following technical changes:
+
+.Federal Information Processing Standard (FIPS)
+
+Federal Information Processing Standards (FIPS) are a set of computer security standards developed by the United States federal government in line with the Federal Information Security Management Act (FISMA).
```

Comment:
@stillalearner & @rayfordj - should FIPS be a **new feature** or a **known issue**?
thanks

Reply:
While I find "technical changes" fitting, of those two choices, "**new feature**"

Reply:
i like FIPS in technical changes

@stillalearner - can you live with this?